File: beginners/ex6.py
A, op, B = input().split()
A = int(A)
B = int(B)

if op == "+":
  print(A + B)

# ここにプログラムを追記
elif op == "-":
  print(A - B)
elif op == "*":
  print(A * B)
elif op == "/":
  if B != 0:
    print(A // B)
  else:
    print("error")
else:
  print("error")

File: beginners/ex6.cpp
#include <iostream>
using namespace std;

int main() {
  int A, B;
  string op;
  cin >> A >> op >> B;

  if (op == "+") {
    cout << A + B << endl;
  }

  if (op == "-") {
    cout << A - B << endl;
  }

  if (op == "*") {
    cout << A * B << endl;
  }

  if (op == "/") {
    if (B == 0) {
      cout << "error" << endl;
    } else {
      cout << A / B << endl;
    }
  }

  if (op == "?" || op == "=" || op == "!") {
    cout << "error" << endl;
  }

  return 0;
}

